website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-okta-deprecated.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Setting up SSO with Okta (Deprecated)"
+id: "setting-up-sso-with-okta-deprecated"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
+
+:::
+
+## Okta SSO
+
+dbt Cloud Enterprise supports single-sign on via Okta (using SAML). Currently supported features include:
+
+* IdP-initiated SSO
+* SP-initiated SSO
+* Just-in-time provisioning
+
+This guide outlines the setup process for authenticating to dbt Cloud with Okta. After following the steps below, please contact support (support@getdbt.com) to complete the setup process.
+
+## Configuration
+
+### Create a new application
+
+Log into your Okta account. You'll need administrator access to your Okta organization to follow this guide.
+
```

Comment:
I poked around Okta a little - doesn't look like there's an easy link to provide here that would generalize to a list of admins on a given company's subdomain. Would have been a nice touch 😅

Reply:
I thought about adding some inputs to the top of the page for:
 - your Okta domain
 - your dbt Cloud domain

And we could use those inputs to render specific docs for different users :). Let's do that in the v2!